test/spec/endpoints.test.js

```diff
@@ -10,23 +10,34 @@ let res = {
   messages: []
 }
 
-// nock('https://slack.com').post('/api/chat.delete').times(1).reply(200, res)
-nock('https://slack.com')
-  .filteringPath(function (path) {
-    return path.match(/\/api\/channels.history/)
+describe('/ephemeral', function () {
+  before(function () {
+    // nock.recorder.rec()
+    nock.enableNetConnect(/^.*127\.0\.0\.1.*$/)
+
   })
-  .get('/api/channels.history').times(1).reply(200, res)
 
-describe('/ephemeral', function () {
-  before(() => nock.enableNetConnect('127.0.0.1'))
-  after(() => nock.disableNetConnect('127.0.0.1'))
+  beforeEach(function () {
+    nock('https://slack.com')
+      .filteringPath(function (path) {
+        return path.match(/\/api\/channels.history/)
+      })
+      .get('/api/channels.history').times(1).reply(200, res)
+  })
+
+  after(function () {
+    // nock.recorder.play()
+    nock.disableNetConnect(/^.*127\.0\.0\.1.*$/)
+  })
+
+  afterEach(function () {
+    nock.cleanAll()
```

Comment:
This should unglue nocks that don't work, so if a test fails, we're not still waiting for a response.